environments/common/application-load-balancer/locals.tf

```diff
@@ -0,0 +1,44 @@
+locals {
+  services = {
+    admin = {
+      target_group_name = "trade-tariff-ad-tg-${var.environment}"
+      priority          = 50
+      paths             = ["/admin/*"] # TODO: confirm routes
+    }
+
+    backend = {
+      target_group_name = "trade-tariff-be-tg-${var.environment}"
+      priority          = 10
+      paths = [
+        "/chapters/*",
```

Comment:
I could do with a run down of ALB functionality/concepts at some point. These paths don't feel quite right to me.

Reply:
No problem. We'll have to get together and work out which apps we can route like this and where the edge cases are